src/main/java/org/codewars/utils/TaskRunner.java

```diff
@@ -1,108 +1,127 @@
 package org.codewars.utils;
 
 import java.math.BigInteger;
+import java.util.InputMismatchException;
 
 public class TaskRunner {
-    public static void setUser(int id){
+
+    private final Users user;
+
+    public static void setUser(int id) {
         // TODO: Implement setUser() method
     }
+
     public static long[] runGap() {
         return new long[0];
     }
 
-    public static BigInteger runPerimeter(){
+    public static BigInteger runPerimeter() {
         return new BigInteger("0");
     }
 
-    public static int runZeros(){
-       return 0;
-    }
-
-    public static double runF(){
-        return 0.0;
-    }
-
-    public static long runFindNb(){
+    public static int runZeros() {
         return 0;
     }
 
-    public static double runMean(){
-        return 0;
+    public static double runF() {
+        return 0.0;
     }
 
-    public static double runVariance(){
+    public static long runFindNb() {
         return 0;
     }
 
-    public static String runBalance(){
+    public void rainfall() {
+        try {
+            System.out.println("Run task Rainfall: mean method");
+            System.out.println("Input the name of the city");
+            String town = Reader.readString();
+            System.out.println("Input string with rainfall records. City entries should be separated by \\n." +
+                    "The name of each city is followed by :");
+            String str = Reader.readString();
```

Comment:
mb you've create branch from old dev? now Reader uses by reader (initializes in constructor)

Reply:
If I'm not mistaken, Lubomyr recommended renaming the branches